repos/cli.proj

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommandArgs>--stage0 $(DotNetCliToolDir)</BuildCommandArgs>
+    <RuntimeArg Condition="'$(TargetOS)' != 'OSX'">--runtime-id $(TargetRid)</RuntimeArg>
```

Comment:
Why is OSX special? 

I suspect windows maybe special in the same way.

Reply:
I believe you are correct.  The relevant code sets the `HostMonikerRid` [here](https://github.com/dotnet/cli/blob/4db4d9b568d575b379e2b9174d1aeba093695c07/build/Branding.props#L20).  Then `CoreSetupRid` gets set to `HostMonikerRid` [here](https://github.com/dotnet/cli/blob/4db4d9b568d575b379e2b9174d1aeba093695c07/build/BundledRuntimes.props#L4).  Then `SharedFrameworkRid` gets set to `CoreSetupRid` [here](https://github.com/dotnet/cli/blob/4db4d9b568d575b379e2b9174d1aeba093695c07/build/BundledRuntimes.props#L27).  Then `SharedFrameworkRid` gets used in filenames like [this](https://github.com/dotnet/cli/blob/4db4d9b568d575b379e2b9174d1aeba093695c07/build/BundledRuntimes.props#L30).  I think restoring the forced Portable=true and special casing our [CLI path](https://github.com/crummel/dotnet_source-build/blob/b97922f0560dff72145295c1466ee9fa15632f3b/repos/cli.proj#L32) for Windows and Mac vs Linux is probably the best move for now.

cc @livarcocc